src/components/ui/Animal/Animal.stories.js

```diff
@@ -0,0 +1,91 @@
+import Animal from '../../Animal/Animal';
+
+export default {
+	title: 'Components/Animal',
+	component: Animal,
+	decorators: [
+		Story => {
+			return (
+				<div style={{ padding: '3em', maxWidth: '400px' }}>
+					<Story />
+				</div>
+			);
+		},
+	],
+};
+
+const props = {
+	spots: [
+		{
+			id: '1',
+			name: 'Bimini',
+			country: 'Bahamas',
+			geoTag: { latitude: '25.703188', longitude: '-79.281459' },
+			description: 'nice and warm',
+		},
+		{
```

Comment:
Since you are using the same fake data here as well as in your Animal.spec.js, why not outsourcing this variable to another file and simply import it in bot spec files?

Reply:
Unfortunately it's not the same data.